pkg/ottl/contexts/internal/ottlcommon/span.go

```diff
@@ -84,8 +84,20 @@ func SpanPathGetSetter[K SpanContext](path []ottl.Field) (ottl.GetSetter[K], err
 		return accessKind[K](), nil
 	case "start_time_unix_nano":
 		return accessStartTimeUnixNano[K](), nil
+	case "start_time_unix_milli":
+		return accessStartTimeUnixMilli[K](), nil
+	case "start_time_unix_micro":
+		return accessStartTimeUnixMicro[K](), nil
+	case "start_time_unix_sec":
+		return accessStartTimeUnixSec[K](), nil
 	case "end_time_unix_nano":
 		return accessEndTimeUnixNano[K](), nil
+	case "end_time_unix_milli":
+		return accessEndTimeUnixMilli[K](), nil
+	case "end_time_unix_micro":
+		return accessEndTimeUnixMicro[K](), nil
+	case "end_time_unix_sec":
+		return accessEndTimeUnixSec[K](), nil
```

Comment:
Instead of doing this, what about we do smarter in parsing, and treat "start_time" as `time.Time` so people can write `where start_time < ANY_TIME_TIME_FORMAT`

Reply:
or something like start_time < time("...", "...") where time func matches the definition of https://pkg.go.dev/time#Parse?

Reply:
I like the idea of using native time, it'll make lots of interactions after Getting easy, and it will allow consistent setting (because it will always take a Time). 

It strays from the notion that the contexts are one-to-one with the protobuf fields, but I think the benefits will be worth it.